src/dsp_tools/commands/ingest_xmlupload/ingest_files/ingest_files.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+from pathlib import Path
+from time import sleep
+
+from loguru import logger
+
+from dsp_tools.cli.args import ServerCredentials
+from dsp_tools.commands.ingest_xmlupload.bulk_ingest_client import BulkIngestClient
+from dsp_tools.utils.connection import Connection
+from dsp_tools.utils.connection_live import ConnectionLive
+
+
+def ingest_files(creds: ServerCredentials, shortcode: str) -> bool:
+    """
+    Kick off the ingest process on the server, and wait until it has finished.
+    Then, retrieve the mapping CSV from the server and save it in the CWD.
+
+    Args:
+        creds: credentials to log in on the server
+        shortcode: shortcode of the project
+
+    Returns:
+        success status
+    """
+    con: Connection = ConnectionLive(creds.server)
+    con.login(creds.user, creds.password)
+    bulk_ingest_client = BulkIngestClient(creds.dsp_ingest_url, con.get_token(), shortcode)
+    bulk_ingest_client.kick_off_ingest()
+    sleep(5)
+
+    while not (mapping := bulk_ingest_client.retrieve_mapping()):
```

Comment:
you could use tqdm to indicate that it's still working, how often it has already polled and how quickly it does so - obviously not with a real progress bar (so it won't look as good), but it might be better than just no output at all.  
For how to do it, see https://stackoverflow.com/a/67249498/11914562 or https://stackoverflow.com/a/73319148/11914562

Reply:
> but it might be better than just no output at all

I can look at you 2 links after my holidays, and if possible, I will gladly implement something with tqdm. But there is already a print output now.

Reply:
Done with [d86d04f51f9328f2fe0c03f5c51925047a8f1725](https://github.com/dasch-swiss/dsp-tools/commit/d86d04f51f9328f2fe0c03f5c51925047a8f1725)